bootcamp/RAG/zilliz_bento_RAG.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.               for embedding in embedding_split:
</pre><ol><li>make sure <code>embedding_split</code> is a list of embeddings</li><li>replace line 9 &amp; 10 with <code>embeddings += embedding_split</code>   </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
fixed